install.sh

```diff
@@ -2,34 +2,34 @@
 
 # Source setting.sh
 source /vagrant/settings.sh
+
+# Fetching LAN_IP and network address
+if [[ ${STANDALONE,,} != yes ]]; then
+   LAN_IP=$(ip addr | grep "inet\b" | cut -d" " -f6 | grep 192 |  cut -d/ -f1)
+   sed -i /vagrant/settings.sh \
+      -e "/LAN_IP=""/ c LAN_IP=\"$LAN_IP\""
```

Comment:
It is not a good idea to modify `settings.sh` from `install.sh`.
I also don't think you need to do that.